package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "pug-addon",
   "description": "Augmented Reality Game Experience",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
You changed the package.json `version`, but do we also need to modify the addon/webextension/manifest.json `version` string?